Fixes Bug 1537890

DraciVik opened the pull request:
"Removed text-align: center in perf.css so that numbers are left alligned by default and added thousands separator for numbers in helpers.js"
Hey @camd @sarah-clements. This is actually my first pull request for open source. Please tell me what I need to fix. Part of this bug was to fix the numbers not having a thousands separators so I changed the helper function "displayNumber" and because it returns a string now with thousands separators it does not pass the test provided for this function.
Is this a problem I need to fix and any directions for it?
All criticism is welcome.

sarah-clements commented on ui/perfherder/helpers.js:
```diff
@@ -22,7 +22,9 @@ import {
-  Number.isNaN(input) ? 'N/A' : Number(input).toFixed(2);
+  Number.isNaN(input)
+    ? 'N/A'
+    : (Math.round(Number(input) * 1e2) / 1e2).toLocaleString();
```
This looks like it works well but you also use `Intl.NumberFormat` without needing to round and divide the number (it's used elsewhere in the codebase).

sarah-clements commented:
This is a good start but you will need to update the failing test to get it to pass. 

If you click on the red x that says "continuous-integration/travis-ci/pr — The Travis CI build failed" (above), it will show you which test suite is failing. Click into that and you'll see what failed. 

You should also follow all of the tips outlined in the Project Contribution Information section on the Outreachy project page. It has links to our documents for testing and formatting code. Read through those and run the commands in your terminal to see what is failing and fix it in the test.

DraciVik commented:
Hey @sarah-clements. Sorry for the delay, but I had a volunteer project to organize and execute in the past 2 days. 
Here I added a formatNumber function in helpers.js that uses Intl.NumberFormat to format a number provided as an argument. 
I also added tests for this function in the helpers_test.js file similar to those of the displayNumber function from the helpers.js file.
All tests pass on my end.
If everything is OK I would like to proceed to the next phase.
Thank you!

sarah-clements commented on tests/ui/helpers/helpers_test.js:
```diff
@@ -17,3 +17,12 @@ describe('displayNumber helper', () => {
+    expect(formatNumber(1 / 0)).toEqual('∞');
+    expect(formatNumber('a string')).toEqual('NaN');
+  });
+});
```
Adding a test here shows really good initiative but I don't think it's necessary since it's testing a built-in Javascript method, rather than custom logic. So I think you can remove it (I'm not sure why someone added a test for `displayNumber` but we can leave that for now).

sarah-clements commented on ui/perfherder/helpers.js:
```diff
@@ -21,6 +21,12 @@ import {
 } from './constants';
 
+export const formatNumber = input => {
+  return new Intl.NumberFormat('en-US', { maximumFractionDigits: 2 }).format(
```
Since it's only one line of code (even though its formatted to be multi-line) you can remove the curly brackets and the return statement like so: 
```
export const formatNumber = input => new Intl.NumberFormat('en-US', { maximumFractionDigits: 2 }).format(
    input,
  );
```

sarah-clements commented:
> Hey @sarah-clements. Sorry for the delay, but I had a volunteer project to organize and execute in the past 2 days.
> Here I added a formatNumber function in helpers.js that uses Intl.NumberFormat to format a number provided as an argument.
> I also added tests for this function in the helpers_test.js file similar to those of the displayNumber function from the helpers.js file.
> All tests pass on my end.
> If everything is OK I would like to proceed to the next phase.
> Thank you!

Yes, you can start on the next task.
